src/auth/authApp.ts

```diff
@@ -55,26 +57,10 @@ export function isUserSignedIn() {
 }
 
 /**
- * Generates an authentication request and redirects the user to the Blockstack
- * browser to approve the sign in request.
- *
- * Please note that this requires that the web browser properly handles the
- * `blockstack:` URL protocol handler.
- *
- * Most applications should use this
- * method for sign in unless they require more fine grained control over how the
- * authentication request is generated. If your app falls into this category,
- * use `makeAuthRequest` and `redirectToSignInWithAuthRequest` to build your own sign in process.
- *
- * @param {String} [redirectURI=`${window.location.origin}/`]
- * The location to which the identity provider will redirect the user after
- * the user approves sign in.
- * @param  {String} [manifestURI=`${window.location.origin}/manifest.json`]
- * Location of the manifest file.
- * @param  {Array} [scopes=DEFAULT_SCOPE] Defaults to requesting write access to
- * this app's data store.
- * An array of strings indicating which permissions this app is requesting.
- * @return {void}
+ * @deprecated v19
+ * 
+ * Use [[UserSession.redirectToSignIn]] instead.
+ * 
  */
 export function redirectToSignIn(redirectURI?: string, 
```

Comment:
These `@deprecated` tags and reference links are really handy! I think we use these, but also keep the desc and param descriptions intact until we have the rest of tutorials/docs, app generators, and e2e tests changed to using the `UserSession` class. 

And maybe keep them until/unless we remove the functions all-together, since the available params and behavior between these functions and the `UserSession` versions may not be the same, or diverge, either intentionally or due to bugs.

Reply:
@zone117x I think that we should just remove them and leave the deprecated. The generated files have links to the code if they are using the existing call and want to maintain it -- a single click.  And the link to the `UserSession` is there if they got here incorrectly or want to go to the latest.

Reply:
Two main issues I'm concerned about here:

1. `UserSession` versions of these functions are not covered in our integration tests, and could have unknown or undocumented behavioral differences from the original. This has already happened with several of these functions. Leaving docs blank and linking to the other version implies that this is will not be the case. 

2. The `UserSession` versions can (and already have) intentionally diverged in usage & behavior from the original ones. For example, the current docs for `UserSession.redirectToSignIn(...)`, copied over from the original, are not really accurate. It tells devs to use other class methods (like `makeAuthRequest`) if they `require more fine grained control over how the authentication request`. Once updated to explain that class constructor options are the intended way to customize auth flow options, these empty docs with links to UserSession are _less_ than helpful, and will force devs to read through src to figure out what's going on. 

Removing these functions from the docs entirely would cause devs less problems and confusion, but we can't yet do that because all our other material still points devs to these functions. 

Seems reasonable to leave the existing, accurate docs for these intact, then we can eventually start removing things once they are effectively deprecated, rather than just tagged as such.

Reply:
1. Isn't this a problem with how `UserSession` and the tests were implemented? If use of `UserSession` is problematic **we should fix the problems for `UserSessions`** . We deprecated the old functions, we say they are deprecated, we should not be documenting them. But if we haven't safely implemented the new functions why did we deprecate the old?

2. The fact that the UserSessions diverged seem fine to me --- they are new and the old ones are deprecated. Again, the problems with the `UserSession` docs should be *fixed if they are the recommended path forward* not left in place with problems we know exist.

Reply:
OK. I added back in the content for the deprecated methods.